### DISCUSSION TITLE
FreshRSS docker container - Extensions do not show up

### DISCUSSION THREAD

**Discussion by chrislongros** [FreshRSS/FreshRSS#7016] [Q&A]
### Describe the bug

I run a docker container and copied the github repository of Extensions (https://github.com/FreshRSS/Extensions) to the /var/www/FreshRSS/extensions that I created. Then I rerun:

sudo docker run -d --restart unless-stopped --log-opt max-size=10m \
  -p 8080:80 \
  -e TZ=Europe/Paris \
  -e 'CRON_MIN=1,31' \
  -v freshrss_data:/var/www/FreshRSS/data \
  -v freshrss_extensions:/var/www/FreshRSS/extensions \
  --name freshrss \
  freshrss/freshrss

However the extensions do not show up: I only see User CSS and User JS

### To Reproduce

Install the docker image and copy the extensions repository to the /var/www/FreshRSS/extensions directory

### FreshRSS version

1.24.3

### Environment information

- Database version: [e.g. Mysql 5.7, SQLite]
- PHP version: 8.3.14-1
- Installation type: Docker image from https://hub.docker.com/r/freshrss/freshrss
-Web server type: Apache 2.4.62-1
- OS: Arch Linux
- Browser: Firefox 133.0 (64-bit)

**Comment by Alkarex** [comment]
Hello,
How did you copy the extensions? `/var/www/FreshRSS/extensions/` is inside the Docker container, not directly on your machine.

Alternatively, you could change the left side of `-v freshrss_extensions:/var/www/FreshRSS/extensions` to point to a local directory on your computer like `-v /home/chrislongros/freshrss-extensions:/var/www/FreshRSS/extensions`

Finally, you would have a more robust and reproducible setup using Docker Compose. See
https://github.com/FreshRSS/FreshRSS/blob/edge/Docker/README.md#docker-compose

**Comment by chrislongros** [comment]
I tried pointing to a local directory and did the setup again. I saw no available extensions in the extension section...

When I try `sudo docker exec -it freshrss sh` I can not use wget or git to pull the extension files:

```
wget https://github.com/FreshRSS/Extensions/archive/refs/heads/master.zip
sh: 3: wget: not found
```

**Reply by Alkarex** [reply]
`apt update && apt install curl wget`

**Comment by chrislongros** [comment]
So I managed to install git using a debian container and the extensions work !!
